docs/admin/sso/saml.md

```diff
@@ -15,6 +15,41 @@ To fully configure SAML SSO, you will need to generate a configuration in FlowFu
 provide some of the generated values to your Identity Provider, and copy back some
 values they provide.
 
+## Configuring SSO on FlowFuse Cloud
+
+Configuring SSO on FlowFuse Cloud requires co-ordinating tasks between the customer and
+FlowFuse Cloud administrators.
+
+When a customer requests SSO to be setup for their users, we require the following information:
+
+1. The email domain that will be covered by the configuration. Note that each SSO configuration can only be applied to a single domain. If a customer has multiple domains, each one will require its own SSO configuration. [Issue #5011](https://github.com/FlowFuse/flowfuse/issues/5011) has been raised to make this more flexible in the future.
+2. Whether it is SAML or LDAP based SSO
+3. What Identify Provider they are using for their SSO.
+
+Once this information has been provided, we can then create a draft SSO configuration in the Admin/Settings/SSO section. The configuration should not be marked as active yet.
+
+From the draft configuration, the values of `ACS URL` and `Entity ID / Issuer` can be given to the customer. These values will be required by their Identify Provider. Refer to the provider-specific documentation for how those values get applied.
+
+In return, the customer then needs to provide:
+
+1. `Identity Provider Single Sign-On URL`
+2. `Identity Provider Issuer ID / URL`
+3. `X.509 Certificate Public Key`
+
+Again, refer to the provider-specific documentation for where to find these values as each provider has its own terminology.
+
+These values should be applied to the draft SSO configuration in FlowFuse Cloud. The configuration can then be marked as active.
+
+It is recommended to do this final step whilst on a call with the customer so they can test the setup in real time.
```

Comment:
This appears to be specific to SAML SSO, the setup for LDAP would be different (e.g. needing LDAP creds and search DN values.

Reply:
Yup - its on the SAML-only docs